src/shared/components/palette/judicial-user/read-judicial-user-field.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { JudicialUserModel } from '../../../domain/jurisdiction';
+import { JurisdictionService } from '../../../services';
+import { AbstractFieldReadComponent } from '../base-field';
+
+@Component({
+  selector: 'ccd-read-judicial-user-field',
+  templateUrl: './read-judicial-user-field.component.html'
+})
+export class ReadJudicialUserFieldComponent extends AbstractFieldReadComponent implements OnInit, OnDestroy {
+
+  public judicialUser: JudicialUserModel;
+  public sub: Subscription;
+
+  constructor(private readonly jurisdictionService: JurisdictionService) {
+    super();
+  }
+
+  public ngOnInit(): void {
+    const personalCode = this.caseField.value.personalCode;
```

Comment:
@johnbenjamin-hmcts - should there be a check for `this.caseField.value`? What happens if `caseField` doesn't have a `value`?

Reply:
Thank you @Daniel-Lam - I have done the changes.